src/status.rs

```diff
@@ -49,6 +49,7 @@ pub struct State {
 #[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
 pub struct Event {
     pub name: String,
+    #[serde(rename = "type")]
     pub _type: String,
```

Comment:
In line with https://github.com/rust-lang-nursery/fmt-rfcs/issues/84#issuecomment-309050802, please change the name to `type_`.